k9mail/src/main/java/com/fsck/k9/helper/MessageHelper.java

```diff
@@ -179,7 +182,7 @@ public static CharSequence toFriendly(Address[] addresses, Contacts contacts) {
                 }
             }
         }
-
+        if(showCorrespondentNamesAndEmailAdresses) return (!TextUtils.isEmpty(address.getPersonal())) ? address.getPersonal() + " <" + address.getAddress() + ">" : address.getAddress();
```

Comment:
This line is really long. Also the logic is a bit convoluted - a return containing a ! and a ternary